src/Elasticsearch/Helper/Iterators/SearchHitIterator.php

```diff
@@ -63,6 +70,8 @@ public function rewind()
             $this->search_responses->next();
         }
 
+        $this->count = $current_page['hits']['total'];
```

Comment:
Perhaps we should put an `isset()` conditional around this to protect against unforeseen errors in the response and the `['hits']['total']` isn't set?  Just default to zero otherwise?